pronto/api/checks/annotations.py

```diff
@@ -15,6 +15,22 @@
 Err = List[Tuple[str, Optional[str]]]
 LoT = List[Tuple[str, str]]
 
+def ck_begin_uppercase(cabs: LoT, exceptions: DoS) -> Err:
+    prog = re.compile(r"^<p>[a-z]")
+    prog2 = re.compile(r"^[a-z]")
+
+    errors = []
+    for ann_id, text in cabs:
+        for match in prog.findall(text):
+            errors.append((ann_id, match))
+        for match in prog2.findall(text):
+            errors.append((ann_id, match))
```

Comment:
I don't think it is necessary to call `re.findall()`: since we looking for at the beginning of the string (`^` in the pattern), only one match can be found. 

Also, we can use non-capturing parentheses to only compile one regular expression only and allow optional spaces before the first letter.

Finally, we can report the lower case letter _and_ the following characters, to give more context (if we take the lower case letter and the 10 following characters, the error would be `this family` instead of `t`).

```suggestion
    prog = re.compile(r"^\s*(?:<p>)?\s*([a-z])")

    errors = []
    for ann_id, text in cabs:
        match = prog.match(text)
        if match:
            start = match.start(1)
            error = text[start:start+11]
            errors.append((ann_id, error))
```